core/src/epicli/data/common/ansible/playbooks/upgrade.yml

```diff
@@ -99,6 +99,7 @@
     - import_role:
         name: upgrade
         tasks_from: elasticsearch-curator
+      when: "'elasticsearch_curator' in upgrade_components or upgrade_components|length == 0"
```

Comment:
```suggestion
```
Selective upgrades is a new feature, probably not present in v0.8.

Reply:
Gotcha! I removed this condition.